file-formats/ntty/type/zif_aff_ntty_v1.intf.abap

```diff
@@ -19,8 +19,8 @@ INTERFACE zif_aff_ntty_v1
       "! <p class="shorttext">Multiple Notes</p>
       "! Multiple notes possible
       multiple_notes            TYPE abap_bool,
-      "! <p class="shorttext">Hide Title</p>
-      "! Hides the title
+      "! <p class="shorttext">Title hidden</p>
+      "! Title hidden
       hide_title          TYPE abap_bool,
```

Comment:
In order to align the ABAP variable and JSON field name to the UI text in the editor, you might consider to rename here accordingly. A simple rename in ADT should do the work for you ;)

```suggestion
      title_hidden          TYPE abap_bool,
```

See, the checkboxes in the UI 'Language Dependent' and 'Multiple Notes' correspond to 'language_dependent' (abap) or 'languageDependent' (json example). Same for 'Multiple Notes' but not for 'Title Hidden'.

To be clear, this is not a blocking issue im my opinion, but definitively a worthy change now.

Reply:
Thank you for the hint. I also noticed it and if I change it I would also need to change it in the database table. Otherwise I have to map it everytime. This is unfortunately not only a rename with ADT :(